packages/client/src/runtime/index.ts

```diff
@@ -18,21 +22,17 @@ export {
   MetricsClient,
 } from './core/metrics/MetricsClient'
 export { defineDmmfProperty } from './core/runtimeDataModel'
+export * from './core/types/exported'
+export type { ITXClientDenyList } from './core/types/exported/itxClientDenyList'
 export { objectEnumValues } from './core/types/exported/ObjectEnums'
 export type { PrismaClientOptions } from './getPrismaClient'
 export { getPrismaClient } from './getPrismaClient'
 export { makeStrictEnum } from './strictEnum'
 export { warnEnvConflicts } from './warnEnvConflicts'
 export { Debug } from '@prisma/debug'
 export type { DriverAdapter } from '@prisma/driver-adapter-utils'
+export { warnOnce } from '@prisma/internals'
 export { default as Decimal } from 'decimal.js'
+export { detectRuntime } from 'detect-runtime'
```

Comment:
We should sooner or later upgrade here:
> Warning detect-runtime has been deprecated in favor of [std-env](https://github.com/unjs/std-env) (since version 3.4.0), which includes the same functionalities and much more.
via https://github.com/QuiiBz/detect-runtime

Reply:
https://github.com/prisma/team-orm/issues/626

Reply:
Thanks for the reminder